src/mitochondria/transactions.ts

```diff
@@ -1,29 +1,16 @@
-import { TransactionType } from '../declarations/transaction';
-import { IAuthState } from '../store/reducers/auth';
 import {post} from './';
+import { ITransaction } from './../declarations/transaction';
+import { INewTransaction } from './transactions';
 
 /**
  * Interface for body of a post request (Add Transaction) of the API
  */
-export interface INewTransaction {
-  company_id: number;
-  money: number;
-  description: string;
-  date: string;  // ISO 8601 format: 'YYYY-MM-DD'
-  type: TransactionType;
-  notes?: string;
-}
+export type INewTransaction = Omit<ITransaction, 'id'|'recurring_id'>;
 
 /**
  * This function call the calls the @function post function to create the Transaction in the database.
  * @param transaction The transaction we want to post to the API
- * @param authState The authentication state for authenticating with the API
  */
-export const createTransaction = async (transaction: INewTransaction, authState: IAuthState) => {
-  try {
-    const response = await post('/transaction', transaction, authState);
-    return response;
-  } catch (e) {
-    throw e;
-  }
+export const createTransaction = async (transaction: INewTransaction) => {
+    return await post('/transaction', transaction);
```

Comment:
If I understand this code correctly, then this does not handle expired/invalid tokens, right?

Reply:
I'll try to finish up and implement #73 so we have a generic `fetch` method called `fetchWithToken` that can be used for calls that need to be authorized.

Reply:
> I'll try to finish up and implement #73 so we have a generic `fetch` method called `fetchWithToken` that can be used for calls that need to be authorized.

@FredrikAugust 	Which use cases do we make API calls without authorization?